source/kubernetes/manage-app/create-new-env/index.html.md

```diff
@@ -144,6 +139,10 @@ You must be in the office or on the VPN to access this endpoint.
 
 If the environment URL endpoint is not behaving as expected or shows an error, contact [#govuk-platform-engineering team].
 
+## Setting access permissions
+
+You may want to ensure that the correct engineers have access to the account/environment you have just set up. You can check and configure this in the [govuk-user-reviewer](https://github.com/alphagov/govuk-user-reviewer) repository.
```

Comment:
I noticed we have a private repository here. Maybe we can put the repository URL for GDS AWS accounts?